.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: Tests
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  windows:
+    runs-on: windows-latest
+    env:
+      DUMMY_CONVERSION: True
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - run: pip install poetry
+      - run: poetry install
+      - name: Run CLI tests
+        run: poetry run pytest -v
```

Comment:
Quick question, why don't we use `make test` here? I think it should be available in the CI runner.

Reply:
By default windows doesn't have `make`. But I can try and see if it works.

Reply:
You're right! Make is available! https://github.com/freedomofpress/dangerzone/actions/runs/3948344493/jobs/6758222698

Closing this.